packages/repco-webcomponent/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
as above: let's keep a single setup as with the other packages (config in root of repo)